src/Components/Card/Card.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from "react";
+import { CategoryContext } from "../../Context/category-context";
+import { Link } from "react-router-dom";
+import { categories } from "../../backend/db/categories";
+import "./Card.css";
+
+export default function Card() {
+  const { setCategory } = useContext(CategoryContext);
+  return (
+    <div className="card-container">
+      {categories.map((cat) => (
+        <div key={Math.random()} className="card">
+          <div className="background-Image">
+            <img
+              className="card-img"
+              src="/assets/images/bg1.jpg"
```

Comment:
You can import images first and then directly apply them to the src attribute. [refer this](https://stackoverflow.com/questions/43823289/how-to-import-image-svg-png-in-a-react-component)

Reply:
I tried but I was facing some issues so I removed it.